cmd/fmt_test.go

```diff
@@ -176,14 +176,14 @@ p {
 	test.WithTempFS(files, func(path string) {
 		policyFile := filepath.Join(path, "policy.rego")
 		info, err := os.Stat(policyFile)
-		err = formatFile(&params, &stdout, policyFile, info, err)
+		err = formatFile(&params, io.Discard, policyFile, info, err)
```

Comment:
Sorry I wasn't explicit here; the same applies to the test below 😅

Reply:
Doh should have caught that on my end, cleaned up now.